controllers/gatekeeper_controller.go

```diff
@@ -397,6 +398,9 @@ func crOverrides(gatekeeper *operatorv1alpha1.Gatekeeper, asset string, obj *uns
 		if err := webhookConfigurationOverrides(obj, gatekeeper.Spec.Webhook, ValidationGatekeeperWebhook); err != nil {
 			return err
 		}
+		if err := webhookConfigurationOverrides(obj, gatekeeper.Spec.Webhook, CheckIgnoreLabelGatekeeperWebhook); err != nil {
```

Comment:
This will also set the `FailurePolicy` to match whatever the user has selected in the `Gatekeeper` CR. I'm not sure that's what we should do. I think we would need to potentially have separate failure policies for both webhooks (`validation.gatekeeper.sh` and `check-ignore-label-gatekeeper.sh`), or not allow the ability to override the failure policy for the `check-ignore-label-gatekeeper.sh` webhook. I don't think I see a reason the `check-ignore-label-gatekeeper.sh` webhook should have an ignore failure policy, which would lead me to prefer the 2nd option. Thoughts?

Reply:
Fixed.

I would not allow it. Probably better to fail some namespace updates, then to let someone unauthorized set of the label.
This makes me also think whether we should not protect the namespace selector set on the webhook by the operator. If someone set's the right labels (pretending to be control plane), the resources in the namespace could be excluded from mutation.

Reply:
> This makes me also think whether we should not protect the namespace selector set on the webhook by the operator. If someone set's the right labels (pretending to be control plane), the resources in the namespace could be excluded from mutation.

Can you elaborate on what you mean by "not protect"